File: blog-angular-backend/src/main/java/fr/malkav/modele/Item.java
package fr.malkav.modele;

public class Item extends ItemLite {

	private String content;

	public Item() {
	}
	
	public Item(int id, String name, Category category, User creator, String imageHeadUrl, String commentaire,
			String content) {
		super(id, name, category, creator, imageHeadUrl, commentaire);
		this.content = content;
	}

	public String getContent() {
		return content;
	}

	public void setContent(String content) {
		this.content = content;
	}

}

File: blog-angular-front/src/app/model/items/item.ts
import { ItemLite } from './item-lite';
import { User } from '.././user';
import { Category } from '../categories/category';

export class Item extends ItemLite {

  content: string;

  constructor(id: number, name: string, category: Category, creator: User, imageHeadUrl: string, commentaire: string, content: string) {
    super(id, name, category, creator, imageHeadUrl, commentaire);
    this.content = content;
  }
}
